### PR TITLE
Add Ti50 AP FW RO verification page

### LINKED ISSUE #158: Document TI50 firmware verification on newer (2023+) devices
See https://www.chromium.org/chromium-os/developer-library/guides/device/ro-firmware-unlock/

Newer 2023+ devices use a form of read only firmware verification.

Following the current documentation, if the user does not flash with a suzyq they will end up with a not usable device

### PR DESCRIPTION
**Pull request by WeirdTreeThing**
Closes #158

### CODE REVIEW

**Comment by WeirdTreeThing** [comment]
Other than the merge conflicts this is fine

**Comment by WeirdTreeThing** [comment]
The merge conflicts mentioned above have now been resolved

File: src/docs/firmware/ti50-ap-ro.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+## Step 1: Verify the device has a Ti50
+
+1. Open VT-2 (`ctrl` + `alt` + `f2` (right arrow)).
+2. Login as `root`.
+3. Run `gsctool -a -I | grep AllowUnverifiedRo`. 
+4. If that command has any output, your device has a Ti50. If you get no output or an error, your device either has a Cr50 or no GSC at all.
+
+## Step 2: Disable RO verification.
+1. Unlock the GSC by running `gsctool -a -o`. TODO: explain the unlocking process better (make a GSC unlocking page)
```
Comment (ninelore) [inline src/docs/firmware/ti50-ap-ro.md:21]: Pending ToDo

File: src/docs/firmware/write-protect.md [hunk 2]
```diff
@@ -11,7 +11,12 @@ You do not have to disable write protect if flashing [RW_Legacy](about.md#rw_leg
...
 Depending on your device, you will need to do **one** of the following:
+
 * Take out a WP screw
 * [Unplug your battery](battery.md)
 * Bridge two jumpers
 * [Use a SuzyQable](suzyq.md) (A cable that enables CCD (Closed Case Debugging))
+
+::: warning
+On devices with a Ti50 security chip (2023+), you will additionally need to [disable RO verification](ti50-ap-ro.md).
+:::
```
Comment (ninelore) [inline src/docs/firmware/write-protect.md:22]: Please add a trailing newline

**Review by ninelore — CHANGES_REQUESTED** [review]

File: src/docs/firmware/ti50-ap-ro.md (hunk 1 above)
Comment (WeirdTreeThing) [reply src/docs/firmware/ti50-ap-ro.md:21]: Whoops, missed that. Looking back at this, I think that change is outside of the scope of the PR. I can open an issue to track this if you want.

Comment (ninelore) [reply src/docs/firmware/ti50-ap-ro.md]: > (...) I can open an issue to track this if you want.
Do that

**Review by ninelore — DISMISSED** [review]

**Comment by MrChromebox** [comment]
one thing to note is that this step isn't needed if you disable WP via SuzyQ and use `ccd reset factory` since that will set all the flags to `always/Y` including `AllowUnverifiedRO`

**Comment by WeirdTreeThing** [comment]
Good catch, I'll add a note for that.

**Review by ethanaobrien — APPROVED** [review]

**Review by ninelore — APPROVED** [review]

**Pull request merged**